jibri/rootfs/defaults/jibri.conf

```diff
@@ -29,6 +40,67 @@ jibri {
         {{ end -}}
       }
       {{ end -}}
+      xmpp {
```

Comment:
Don't re-add this here, there is an xmpp.conf file now with this part of the config. I suppose this is a merge error?